app/queries/admin/gdpr_delete_requests_query.rb

```diff
@@ -0,0 +1,19 @@
+module Admin
+  class GDPRDeleteRequestsQuery
+    QUERY_CLAUSE = "users_gdpr_delete_requests.email ILIKE :search OR " \
```

Comment:
This works quite well.  I have a non-blocking concern.

The table name is an implementation detail of the GDPRDeleteRequest model.  This would be a smidge better as a scope on the GDPRDeleteRequest.  Or you could do the following:

```ruby
QUERY_CLAUSE = "#{GDPRDeleteRequest.table_name}.email ILIKE :search OR " \
               "#{GDPRDeleteRequest.table_name}.username ILIKE :search"
```

Again, this isn't a blocker.